# Data Science Tools and Ecosystem

In this Notebook, we will be summarizing the different Data science tools and Ecosystems

## **Objectives**
- Languages of Data Science
- Libraries for Data Science
- Tools for Data Science

The languages that Data SCientists use are the following (In decreasing popularity):
1. Python
2. R
3. SQL
4. Scala
5. Java
6. JavaScript
7. PHP
8. GO
9. Ruby
10. Visual Basic


The different libraries for Data Science are:
- Scientific Computing Libraries in Python:
 1. Pandas
 2. NumPy

- Visualization:
 1. Matplotlib
 2. Seaborn

- Machine Learning:
 1. Scikit-learn
 2. Keras
 3. TensorFlow
 4. PyTorch
        

|Data Science Tools|
----
|Jupyter Notebook|
|RStudio|
|Apache Spark|

### Below are a few examples of an arithmetic expression in Python:

#### Example of addition and multiplication of integers:

In [2]:
(3*4)+5

17

#### In this example we will convert from minutes to hours by dividing by 60:

If an hour is 60 min, 200 min is:

In [4]:
200/60

3.3333333333333335

So 200 min is equal to aprox. 3.3 hours

## Author
Angélica Varas